src/main/proto/commit.proto

```diff
@@ -17,8 +17,8 @@ message Stats {
 
 // Commit information message.
 message Commit {
-    int32 id = 1;  // Commit id. Hashed initial id.
-    int32 repo_id = 2;  // Repo id. TODO(anatoly): add details on how it is
+    string id = 1;  // Commit id. Hashed initial id.
```

Comment:
The comment does not make it clear what happens. Try: Re-hashed original commit id.